core/matching/cache_matcher.go

```diff
@@ -150,5 +151,6 @@ func (this *CacheMatcher) PreloadCache(simulation *models.Simulation) error {
 		}
 	}
 
+	log.Info("Cache preloaded")
```

Comment:
this log should go into the `if` block where the cache is actually pre-loaded

Reply:
got it... done